Header header
float32 drone_timeuint16 tag
uint16 size
int32 rc_ref_pitch
int32 rc_ref_roll
int32 rc_ref_yaw
int32 rc_ref_gaz
int32 rc_ref_ag
